### ISSUE TITLE
What is it and How to use?

### ISSUE THREAD

**Issue by romanr** [hansalemaos/usefuladb#1]
What is it? a set of commands ? or a script? what "useful" does it provide?  
is there a list of commands and parameters?  
The script that is README, are you supposed to run it? or commands from it?  
  
Thank you.

**Comment by hansalemaos** [comment]
README.MD has a list of all possible commands

**Comment by romanr** [comment]
> README.MD has a list of all possible commands

but how to use? 

```
# blocking_shell
zsh: command not found: blocking_shell
```

```
# adb blocking_shell
adb: unknown command blocking_shell
```

**Comment by hansalemaos** [comment]
have you created an instance?

**Comment by romanr** [comment]
Instance of what? I have adb connected to device.

**Comment by hansalemaos** [comment]
Do you know how to use Python?

**Comment by romanr** [comment]
yes. I know how to use virtualenv. usefuladb in installed via pip.
